### PR TITLE
[PIN-3459]Changed response from 201 to 200

### PR DESCRIPTION
**Pull request by rGregnanin**

### CODE REVIEW

File: src/test/resources/logback-test.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
         <appender-ref ref="CONSOLE" />
     </appender>
 
-    <root level="OFF">
+    <root level="ERROR">
```
Comment (beetlecrunch) [inline src/test/resources/logback-test.xml]: Why this change?

Comment (rGregnanin) [reply src/test/resources/logback-test.xml]: I usually change logback test from OFF to ERROR in case of some test fail.
i forgot to change it back

**Review by TonioGela — APPROVED** [review]

**Review by galales — APPROVED** [review]
